managed/ui/src/components/common/associatedUniverse/AssociatedUniverse.scss

```diff
@@ -0,0 +1,8 @@
+.associated-universe-status {
+  margin: 0;
+  line-height: normal;
```

Comment:
I'm not sure but you can try position absolute here.

Reply:
Same as above.